packages/grid/x-data-grid/src/colDef/gridStringOperators.ts

```diff
@@ -86,7 +86,7 @@ export const getGridStringOperators = (): GridFilterOperator[] => [
 
       return ({ value }): boolean =>
         value != null
-          ? filterItem.value.some((filterValue) => {
+          ? filterItem.value.some((filterValue: GridFilterItem['value']) => {
```

Comment:
Same

Reply:
Just to clarify - no changes required on my side, since it will be changed in #4064?

Reply:
Yes